src/app_main.rs

```diff
@@ -218,8 +217,7 @@ where
             } else {
                 None
             };
-            let mut builder = SceneBuilder::for_scene(&mut self.scene);
-            builder.append(fragment, transform);
+            self.scene.append(fragment, transform);
```

Comment:
`SceneBuilder::for_scene` implicity reset the scene. We'll have to do this manually now: `self.scene.reset()`